tailwind.config.js

```diff
@@ -25,6 +25,19 @@ defineAsset("sailor_otter_telescope", 38);
 defineAsset("santa_rosa", 126);
 defineAsset("social_media_icon", 16);
 
+defineAsset("firebase_logo", 133);
+defineAsset("invoca_logo", 98);
+defineAsset("teladoc_logo", 85);
+defineAsset("coe_logo", 92);
+
+defineAsset("stacks_logo", 58);
+defineAsset("atcompany_logo", 85);
+
+defineAsset("corsair_logo", 35);
+defineAsset("replit_logo", 35);
+defineAsset("axure_logo", 35);
+defineAsset("rstudio_logo", 35);
```

Comment:
I wonder, is this actually necessary? The original reason `defineAsset` was created was so that each component of the island/envelope would be sized correctly when absolutely positioned and overlaid on top of each other. This shouldn't be necessary with how we're positioning the logos, so it would be nice to avoid using the `defineAsset` hack if possible.